docs/contributing.md

```diff
@@ -0,0 +1,142 @@
+# Contributing Guide
+
+
+* [New Contributor Guide](#contributing-guide)
+  * [Ways to Contribute](#ways-to-contribute)
+  * [Find an Issue](#find-an-issue)
+  * [Contact Us](#contact-us)
+  * [Pull Request Lifecycle](#pull-request-lifecycle)
+  * [Development Environment Setup](#development-environment-setup)
+  * [Sign Your Commits](#sign-your-commits)
+  * [Pull Request Checklist](#pull-request-checklist)
+
+Welcome! We are glad that you want to contribute to the ORAS project! 💖
+
+As you get started, you are in the best position to give us feedback on areas of
+our project that we need help with, including:
+
+* Problems found during setting up a new developer environment
+* Gaps in our Quickstart Guide or documentation
+* Bugs in our automation scripts
+
+If anything doesn't make sense, or doesn't work when you run it, please open a
+bug report and let us know!
+
+## Ways to Contribute
+
+We welcome many types of contributions including:
+
+* New features
+* New end-to-end tests
+* Bug fixes
+* Documentation
+* Issue Triage
+* Answering questions on Slack channel
+* Web design
+
+Not everything happens through a GitHub pull request.
+Please come to our [meetings](#come-to-meetings) or [contact us](#contact-us) and let's discuss how we can work
+together.
+
+### Come to Meetings
+
+Absolutely everyone is welcome to come to any of our meetings. You never need an
+invitation to join us. In fact, we want you to join us, even if you don’t have
+anything you feel like you want to contribute. Just being there is enough!
+
+Join the weekly ORAS community meeting at 4:00 PM PST on [Zoom](https://zoom.us/j/8079328631?pwd=UzN0NUlmbmcvTTN5L0o5VUQ2YndyQT09) with ID 807 932 8631.
```

Comment:
Note: weekly => bi-weekly. The meeting cadence changed from once a week to bi-weekly in 2023.

Reply:
```suggestion
Join the bi-weekly ORAS community meeting at 4:00 PM PST on [Zoom](https://zoom.us/j/8079328631?pwd=UzN0NUlmbmcvTTN5L0o5VUQ2YndyQT09) with ID 807 932 8631.
```